Package.swift

```diff
@@ -29,9 +29,11 @@ let package = Package(
         .package(url: "https://github.com/apple/swift-log.git", from: "1.4.0"),
     ],
     targets: [
+        .target(name: "CShims"),
```

Comment:
Given that target names have an annoyingly profound uniqueness requirement, can we namespace this more thoroughly? `CAsyncHTTPClient` would be best.